jujugui/static/gui/src/app/components/header-breadcrumb/header-breadcrumb.js

```diff
@@ -138,9 +138,9 @@ YUI.add('header-breadcrumb', function() {
       @method _generateUserLink
     */
     _generateUserLink: function() {
-      const auth = this.props.authDetails;
-      if (auth && (auth.user || auth.loading)) {
-        return this._buildProfile(auth.loading ? '...' : auth.rootUserName);
```

Comment:
Do we no longer need the ellipsis? I note no changes were made to the tests about this removal so either we didn't test this properly or we didn't use it....Just wondering which :)